tests/python_tests/test_llm_pipeline_static.py

```diff
@@ -66,13 +70,40 @@ def test_generation_compare_with_stateful(generation_config):
     assert ref_out == actual_out
 
 
+@pytest.mark.precommit
+@pytest.mark.nightly
+@pytest.mark.parametrize("config", pipeline_configs)
+def test_pipeline_from_blob(config):
+    prompt = 'What is OpenVINO?'
+    model_path = read_model(get_models_list()[0])[1]
+    blob_path = "compiled_model.blob"
```

Comment:
could you please clean up this blob upon test finalize?

Reply:
sure, done